src/index.js

```diff
@@ -8,6 +10,18 @@ export function validateAttributes(attributes) {
   if (typeof attributes !== 'object') {
     throw new Error('Please provide attributes as an object');
   }
+
+  Object.values(attributes).forEach((attrValue) => {
+    const attrType = typeof attrValue;
+    let pass;
```

Comment:
This var is unused, isn't it? You can add eslint to the project to help with these things.